internal/builder/builder.go

```diff
@@ -525,6 +534,18 @@ func (b *Builder) Save(logger logging.Logger, creatorMetadata CreatorMetadata) e
 		return errors.Wrap(err, "adding run.tar layer")
 	}
 
+	if len(b.buildConfigEnv) > 0 {
+		logger.Debugf("Provided Build Config Environment Variables\n  %s", style.Map(b.env, "  ", "\n"))
```

Comment:
Should this be `b.buildConfigEnv`?

Reply:
i tried the way you suggested but the compiler is giving an error,
@natalieparellano i apologize for having delay on this pr , i have been working on my personal project from past 10 days i think, still the pr is able to override build time environment variables but it wasn't able to set it up for runtime environment variables from this [comment](https://github.com/buildpacks/pack/pull/1926#issuecomment-1759419888) , i am trying to figure out the problem and will soon notify you when i am done :) till then i will keep this pr under draft to avoid confusion, thank you

Reply:
I apologize for missing your other comment @WYGIN - we actually don't need to worry about runtime environment variables as these are a whole separate concern. `/cnb/build-config/env` files are just for build time, so everything is working as expected!